etc/workflows/duplicate-event.xml

```diff
@@ -59,6 +59,14 @@
       </configurations>
     </operation>
 
+    <operation
+      id="start-workflow">
+      <configurations>
+        <configuration key="media-packages">${duplicate_media_package_ids}</configuration>
+        <configuration key="workflow_definition">fast</configuration>
+      </configurations>
+    </operation>
+
```

Comment:
I assume this is only for the review? Could you remove it for the merge?

Reply:
I was going to leave the example in but it be default. Or do we not provide usage examples?

Reply:
I would remove it. In any case, `workflow_definition` should be `workflow-definition`.